model/commitqueue/commit_queue.go

```diff
@@ -275,7 +275,7 @@ func preventMergeForItem(patchType string, versionExists bool, item CommitQueueI
 			return errors.New("merge task doesn't exist")
 		}
 		event.LogMergeTaskUnscheduled(mergeTask.Id, mergeTask.Execution, user)
-		if _, err = mergeTask.Block(user); err != nil {
+		if _, err = mergeTask.SetBlockedPriority(user); err != nil {
```

Comment:
I still think it's overloading the blocked concept.

Reply:
Used "disabled" instead.